2-mandatory/2-currency-code-lookup.js

```diff
@@ -19,6 +19,13 @@ const COUNTRY_CURRENCY_CODES = [
 
 function createLookup(countryCurrencyCodes) {
   // write code here
+      const newObject = {};
+      for (const something of countryCurrencyCodes) {
+        const newKey = something[0];
+        const newValue = something[1];
+        newObject[newKey] = newValue;
+      }
+      return newObject;
```

Comment:
This works well and passes the tests. You could choose some more meaningful variable names to make your code more understandable.